SystemServices/SystemServices.cpp

```diff
@@ -650,6 +650,7 @@ namespace WPEFramework {
             strncpy(rebootParam.requestor, "SystemServices", sizeof(rebootParam.requestor));
             strncpy(rebootParam.reboot_reason_custom, customReason.c_str(), sizeof(rebootParam.reboot_reason_custom));
```

Comment:
The same null termination changes should also apply for "rebootParam.reboot_reason_custom" also.